cmake/RunLint.cmake

```diff
@@ -1,18 +1,27 @@
-get_filename_component(LINT_DIR ${LINT_DIR} ABSOLUTE)
-get_filename_component(LINT_PREFIX ${LINT_DIR} PATH)
+get_filename_component(LINT_DIR "${LINT_DIR}" ABSOLUTE)
+get_filename_component(LINT_PREFIX "${LINT_DIR}" PATH)
```

Comment:
It's not that it's less reliable, it's just that if you get in the habit of always wrapping in quotes, then you may accidentally turn a list into a semicolon-separated string of the entries.  Just blindly wrapping all variable expansions in quotes will lead to that problem.  As a result, I believe it's better to add them when needed (which is rare), and leave them off otherwise.  Not only to avoid the habit, but so that other contributors don't end up having that issue too when they mimic what was done.